sdk/spring/spring-cloud-azure-resourcemanager/src/main/java/com/azure/spring/resourcemanager/implementation/crud/EventHubsCrud.java

```diff
@@ -31,6 +33,10 @@ String getResourceType() {
     @Override
     public EventHub internalGet(Tuple2<String, String> namespaceAndName) {
         try {
+            EventHubNamespace eventHubNamespace = new EventHubNamespaceCrud(this.resourceManager,
+                this.resourceMetadata)
+                .get(namespaceAndName.getT1());
+            Assert.notNull(eventHubNamespace, "The Event Hubs namespace should exist first.");
```

Comment:
I think this is not needed, the will add another HTTP call to this method, and if the eventHubNamespace does not exist, the resource manager should throw exception